app/javascript/listings/components/MessageModal.jsx

```diff
@@ -0,0 +1,84 @@
+import { h } from 'preact';
+import PropTypes from 'prop-types';
+
+const MessageModal = ({
+  currentUserId,
+  message,
+  listining,
```

Comment:
I think you meant listing?

```suggestion
  listing,
```

Reply:
✅